app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/vaultunlock/VaultUnlockViewModel.kt

```diff
@@ -208,6 +208,9 @@ class VaultUnlockViewModel @Inject constructor(
 
     private fun handleUnlockClick() {
         val activeUserId = authRepository.activeUserId ?: return
+
+        if (state.input.isEmpty()) return
```

Comment:
Can this ever happen?
We already disable the button when the text is blank.

```kotlin
        isEnabled = state.input.isNotEmpty(),
```

Reply:
With the introduction of KeyboardAction handling this can occur if Done is pressed on the keyboard when input is empty. 

Comparing behavior with the MAUI app, an error dialog should be displayed stating "The Master password field is required." This change in behavior needs to be approved by our product team, or the existing behavior needs to be modified so that it matches MAUI app behavior. 

@qwexter please extract KeyboardAction related changes to a separate PR so that they can be considered independently of the changes related to input fields taking focus.

Reply:
I move out this behavior from PR.